tools/misc/calc_timeprop_spectrum

```diff
@@ -15,17 +15,24 @@ import argparse
 import numpy as np
 from scipy import constants
 
-USAGE = """Reads output from TD calculation (after kick) and produces
-absorption spectra. Needs mux.dat, muy.dat and muz.dat files in working directory."""
+USAGE = """%(prog)s -d xx
+
+Reads output from TD calculation (after kick) and produces absorption spectra.
+
+Needs mux.dat, muy.dat and muz.dat files in working directory."""
 
 def main():
-    parser = argparse.ArgumentParser(description=USAGE)
-    parser.add_argument("tau", metavar="DAMPING",
+    parser = argparse.ArgumentParser(usage=USAGE)
+    parser.add_argument("-d", "--damping", dest="tau",
                         help="damping constant in fs")
-    parser.add_argument("field", metavar="FIELD",
+    parser.add_argument("-f", "--field", dest="field",
                         help="field intensity in volts/angstrom")
```

Comment:
Please specify argument type (`float`) here. Then you don't need the conversion to floats further down.